share/wake/lib/gcc_wake/pkgconfig.wake

```diff
@@ -33,16 +33,17 @@ export def flattenSysLibs (packages: List SysLib): SysLib =
     def lflags  = mapFlat getSysLibLFlags  packages
     SysLib "" headers objects cflags lflags
 
-target pkgConfigImp args =
+target pkgConfigImp flags pkgs =
     require Some pkgConfig = whichInEnvPath "pkg-config" 
     else None
 
-    def cmdline = pkgConfig, args
+    def cmdline = pkgConfig, flags ++ pkgs
     def addenv list = match (getenv "PKG_CONFIG_PATH")
         Some p = "PKG_CONFIG_PATH={p}", list
         None   = list
     def result =
         makeExecPlan cmdline Nil
+        | setPlanLabel "pkg-config: {pkgs | catWith " "}"
```

Comment:
This is the format I mentioned above.  "What is pkg-config doing?  This list of packages."